### ISSUE TITLE
I couldn't find builded version

### ISSUE THREAD

**Issue by serhatserhatli** [couchbaselabs/query#9]
Hi,
Last week when i was looking download page of couchbase web site, download link for n1ql was appearing but now it looks like removed. Where can i get executable builded version for 32 bit windows.
I am going to prepare a presentation about couchbase at my university including application of n1ql
Thank you.

**Comment by behrad** [comment]
the same for me here... 
we have a 3.0 installed on our servers and wanna enable N1QL, but native built downloads seem to be deprecated

**Comment by geraldss** [comment]
Behrad,

You can install the Couchbase Server 4.0 Beta at couchbase.com/beta, and then point the beta cbq-engine to your current Couchbase servers. This is a temporary solution until you upgrade your data to 4.0.

Best,
Gerald

**Comment by behrad** [comment]
Thank you @geraldss , so the embedded n1ql engine in Couchbase Server 4.0 is full compatible or the same as DP4.

**Comment by geraldss** [comment]
Yes. It is an upgrade to DP4. It should be almost fully backward compatible.
